byudml/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '0.6.8'
+__version__ = '0.6.9'
 __metafeature_version__ = '0.4.5'
 __metafeature_path__ = 'd3m.primitives.metalearning.metafeature_extractor.BYU'
 __imputer_version__ = '0.2.1'
```

Comment:
Do we need to update the metafeature and imputer versions too? If there are any changes there, then we should.

Reply:
I have not needed to make any changes to either of the primitives. I did a bug fix to the imputer a couple weeks ago, but I already did a patch release on `byudml` for that, bumping the imputer version too (see 1ac5a7f216e0f064e9c65f9ea2bddf504c936e4d).

Reply:
The metafeature primitive has a changed dependency now on the metalearn package, so it should probably be bumped before we release.